src/de/jost_net/JVerein/Queries/MitgliedQuery.java

```diff
@@ -438,26 +393,135 @@ else if (sort.equals("Geburtstagsliste"))
     {
       sql += " ORDER BY name, vorname";
     }
-    
+
     Logger.debug(sql);
 
     ResultSetExtractor rs = new ResultSetExtractor()
     {
       @Override
       public Object extract(ResultSet rs) throws RemoteException, SQLException
       {
-        ArrayList<Mitglied> list = new ArrayList<>();
+        ArrayList<Long> list = new ArrayList<>();
         while (rs.next())
         {
-          list.add(
-              (Mitglied) service.createObject(Mitglied.class, rs.getString(1)));
+          list.add(rs.getLong(1));
         }
         return list;
       }
     };
 
-    return (ArrayList<Mitglied>) service.execute(sql, bedingungen.toArray(),
-        rs);
+
+    ArrayList<Long> mitgliederIds = (ArrayList<Long>) 
+        service.execute(sql, bedingungen.toArray(), rs);
+
+    String eigenschaftenString = "";
```

Comment:
Zuweisung `= ""` ist redundant und kann entfernt werden. `""` ist der Standardwert von `Strings`

Reply:
Warum meldet dann Eclipse manchmal dass die Variable "may not have been initialized" ist. Das passt nicht.
Ich merge jetzt die zwei Zeilen dann löst sich die Frage auch.

Reply:
Komisch, IntelliJ meldet mir genau das Gegenteil.